source/includes/reconciliationTemplates.md

```diff
@@ -0,0 +1,231 @@
+---
```

Comment:
Seems like the filename needs to begin with an underscore? https://github.com/lord/slate/wiki/Using-Includes